index.html

```diff
@@ -53,7 +53,7 @@
           <div class="soft-quarter-top font-size-base flush-bottom" data-automation-id="live-message-description">
             Join the live stream hourly, on the hour, from 8am-10pm (EST).
           </div>
-          <a href="/live/stream/" class="btn btn-primary btn-sm push-half-top"
+          <a href="/live/stream?autoplay=true&sound=11" class="btn btn-primary btn-sm push-half-top"
```

Comment:
I don't particularly care, but we could do the following now
`{{ "/live/stream" : media_url }}`

then if we ever change the parameter we can just change it in the filter and not have to find all these references

this would require you to merge development into your branch